DESCRIPTION

```diff
@@ -17,11 +17,9 @@ Imports:
     porcelain (>= 0.1.4),
     processx,
     redux,
-    rrq (>= 0.5.5),
+    rrq (>= 0.5.6),
     utils,
-    webutils,
```

Comment:
What happened here? Why do we drop webutils and zip?

Reply:
This was just cleaning up an R CMD CHECK note, webutils no longer used and zip only used in tests